torchvision/ops/roi_align.py

```diff
@@ -66,6 +68,10 @@ def roi_align(input, boxes, output_size, spatial_scale=1.0, sampling_ratio=-1):
     rois = boxes
     if not isinstance(rois, torch.Tensor):
         rois = convert_boxes_to_roi_format(rois)
+    if torch._C._get_tracing_state():
```

Comment:
Can you add a TODO mentioning that this should be changed in the future? Because as it currently stands, this will not work for backwards with JIT.